android/src/main/java/io/sentry/react/RNSentryPackage.java

```diff
@@ -17,7 +18,7 @@ public class RNSentryPackage extends TurboReactPackage {
   @Override
   public NativeModule getModule(String name, ReactApplicationContext reactContext) {
     if (name.equals(RNSentryModuleImpl.NAME)) {
-      return new RNSentryModule(reactContext);
+      return new io.sentry.react.RNSentryModule(reactContext);
```

Comment:
L: Why do we need to add the package import here? do we have a clash?

Reply:
Thanks, just a typo, removed.